### PR TITLE
Add support for tables rich text contentful

### PR DESCRIPTION
**Pull request by lemoswilson**
Adding support for tables as requested [in this task](https://app.asana.com/0/1200777795870740/1202677556921265/f)

### CODE REVIEW

File: components/rich-text/rich-text.coffee [hunk 1]
```diff
@@ -84,6 +84,10 @@ export default
 			[BLOCKS.HEADING_5]: (node, key, create, next) => @renderTextBlock(node, key, create, next, 'h5')
 			[BLOCKS.HEADING_6]: (node, key, create, next) => @renderTextBlock(node, key, create, next, 'h6')
 			[BLOCKS.PARAGRAPH]: (node, key, create, next) => @renderTextBlock(node, key, create, next, 'p')
+			[BLOCKS.TABLE]: (node, key, create, next) => create('table', { key }, next(node.content, key, create, next))
```
Comment (weotch) [inline components/rich-text/rich-text.coffee:87]: This is optional, but you may want to wrap this `<table>` in a div like we're doing with the WYSIWYG component:

https://github.com/BKWLD/cloak/blob/9a1778d18a25db03666cc68fd55d408c0efe2a6b/components/wysiwyg.coffee#L35-L39

Which would get paired with something like this:

https://gitlab.com/clifbar/clifbar/-/blob/master/library/styles/typography.styl#L26-31

So you can horizontally scroll wide tables.

**Review by weotch — APPROVED** [review]

**Comment by weotch** [comment]
Gimme your npm username and I'll add you to the team there so you can publish your own patch version with this once you merge it.

File: components/rich-text/rich-text.coffee (hunk 1 above)
Comment (lemoswilson) [reply components/rich-text/rich-text.coffee:87]: Nice. I'll update the render render function and add a wrapper. Thanks!

**Comment by lemoswilson** [comment]
My npm username is lemoswilson

**Pull request merged**